Windows ARM: GLPK fails to build

```
    [glpk-5.0.p1]   [spkg-install] /usr/bin/bash ../libtool --tag=CC   --mode=link gcc  -g -O2 -version-info 43:1:3 -export-symbols-regex '^glp_*' -no-undefined  -o libglpk.la -rpath /c/a/passagemath/passagemath/sage-local/lib libglpk_la-amd_1.lo libglpk_la-amd_2.lo libglpk_la-amd_aat.lo libglpk_la-amd_control.lo libglpk_la-amd_defaults.lo libglpk_la-amd_dump.lo libglpk_la-amd_info.lo libglpk_la-amd_order.lo libglpk_la-amd_post_tree.lo libglpk_la-amd_postorder.lo libglpk_la-amd_preprocess.lo libglpk_la-amd_valid.lo libglpk_la-advbas.lo libglpk_la-asnhall.lo libglpk_la-asnlp.lo libglpk_la-asnokalg.lo libglpk_la-ckasn.lo libglpk_la-ckcnf.lo libglpk_la-cplex.lo libglpk_la-cpp.lo libglpk_la-cpxbas.lo libglpk_la-graph.lo libglpk_la-gridgen.lo libglpk_la-intfeas1.lo libglpk_la-maxffalg.lo libglpk_la-maxflp.lo libglpk_la-mcflp.lo libglpk_la-mcfokalg.lo libglpk_la-mcfrelax.lo libglpk_la-minisat1.lo libglpk_la-mpl.lo libglpk_la-mps.lo libglpk_la-netgen.lo libglpk_la-npp.lo libglpk_la-pript.lo libglpk_la-prmip.lo libglpk_la-prob1.lo libglpk_la-prob2.lo libglpk_la-prob3.lo libglpk_la-prob4.lo libglpk_la-prob5.lo libglpk_la-prrngs.lo libglpk_la-prsol.lo libglpk_la-rdasn.lo libglpk_la-rdcc.lo libglpk_la-rdcnf.lo libglpk_la-rdipt.lo libglpk_la-rdmaxf.lo libglpk_la-rdmcf.lo libglpk_la-rdmip.lo libglpk_la-rdprob.lo libglpk_la-rdsol.lo libglpk_la-rmfgen.lo libglpk_la-strong.lo libglpk_la-topsort.lo libglpk_la-weak.lo libglpk_la-wcliqex.lo libglpk_la-wrasn.lo libglpk_la-wrcc.lo libglpk_la-wrcnf.lo libglpk_la-wript.lo libglpk_la-wrmaxf.lo libglpk_la-wrmcf.lo libglpk_la-wrmip.lo libglpk_la-wrprob.lo libglpk_la-wrsol.lo libglpk_la-btf.lo libglpk_la-btfint.lo libglpk_la-fhv.lo libglpk_la-fhvint.lo libglpk_la-ifu.lo libglpk_la-luf.lo libglpk_la-lufint.lo libglpk_la-scf.lo libglpk_la-scfint.lo libglpk_la-sgf.lo libglpk_la-sva.lo libglpk_la-colamd.lo libglpk_la-bfd.lo libglpk_la-bfx.lo libglpk_la-glpapi06.lo libglpk_la-glpapi07.lo libglpk_la-glpapi08.lo libglpk_la-glpapi09.lo libglpk_la-glpapi10.lo libglpk_la-glpapi12.lo libglpk_la-glpapi13.lo libglpk_la-glpios01.lo libglpk_la-glpios02.lo libglpk_la-glpios03.lo libglpk_la-glpios07.lo libglpk_la-glpios09.lo libglpk_la-glpios11.lo libglpk_la-glpios12.lo libglpk_la-glpipm.lo libglpk_la-glpmat.lo libglpk_la-glpscl.lo libglpk_la-glpssx01.lo libglpk_la-glpssx02.lo libglpk_la-lux.lo libglpk_la-alloc.lo libglpk_la-dlsup.lo libglpk_la-env.lo libglpk_la-error.lo libglpk_la-stdc.lo libglpk_la-stdout.lo libglpk_la-stream.lo libglpk_la-time.lo libglpk_la-tls.lo libglpk_la-cfg.lo libglpk_la-cfg1.lo libglpk_la-cfg2.lo libglpk_la-clqcut.lo libglpk_la-covgen.lo libglpk_la-fpump.lo libglpk_la-gmicut.lo libglpk_la-gmigen.lo libglpk_la-mirgen.lo libglpk_la-spv.lo libglpk_la-minisat.lo libglpk_la-avl.lo libglpk_la-bignum.lo libglpk_la-dimacs.lo libglpk_la-dmp.lo libglpk_la-ffalg.lo libglpk_la-fp2rat.lo libglpk_la-fvs.lo libglpk_la-gcd.lo libglpk_la-hbm.lo libglpk_la-jd.lo libglpk_la-keller.lo libglpk_la-ks.lo libglpk_la-mc13d.lo libglpk_la-mc21a.lo libglpk_la-mt1.lo libglpk_la-mygmp.lo libglpk_la-okalg.lo libglpk_la-qmd.lo libglpk_la-relax4.lo libglpk_la-rgr.lo libglpk_la-rng.lo libglpk_la-rng1.lo libglpk_la-round2n.lo libglpk_la-spm.lo libglpk_la-str2int.lo libglpk_la-str2num.lo libglpk_la-strspx.lo libglpk_la-strtrim.lo libglpk_la-triang.lo libglpk_la-wclique.lo libglpk_la-wclique1.lo libglpk_la-mpl1.lo libglpk_la-mpl2.lo libglpk_la-mpl3.lo libglpk_la-mpl4.lo libglpk_la-mpl5.lo libglpk_la-mpl6.lo libglpk_la-mplsql.lo libglpk_la-npp1.lo libglpk_la-npp2.lo libglpk_la-npp3.lo libglpk_la-npp4.lo libglpk_la-npp5.lo libglpk_la-npp6.lo libglpk_la-proxy.lo libglpk_la-proxy1.lo libglpk_la-spxat.lo libglpk_la-spxchuzc.lo libglpk_la-spxchuzr.lo libglpk_la-spxlp.lo libglpk_la-spxnt.lo libglpk_la-spxprim.lo libglpk_la-spxprob.lo libglpk_la-spychuzc.lo libglpk_la-spychuzr.lo libglpk_la-spydual.lo libglpk_la-adler32.lo libglpk_la-compress.lo libglpk_la-crc32.lo libglpk_la-deflate.lo libglpk_la-gzclose.lo libglpk_la-gzlib.lo libglpk_la-gzread.lo libglpk_la-gzwrite.lo libglpk_la-inffast.lo libglpk_la-inflate.lo libglpk_la-inftrees.lo libglpk_la-trees.lo libglpk_la-uncompr.lo libglpk_la-zio.lo libglpk_la-zutil.lo  -lgmp -lm 
    [glpk-5.0.p1]   [spkg-install] libtool: link: /c/mingw64/bin/nm -B  .libs/libglpk_la-amd_1.o .libs/libglpk_la-amd_2.o .libs/libglpk_la-amd_aat.o .libs/libglpk_la-amd_control.o .libs/libglpk_la-amd_defaults.o .libs/libglpk_la-amd_dump.o .libs/libglpk_la-amd_info.o .libs/libglpk_la-amd_order.o .libs/libglpk_la-amd_post_tree.o .libs/libglpk_la-amd_postorder.o .libs/libglpk_la-amd_preprocess.o .libs/libglpk_la-amd_valid.o .libs/libglpk_la-advbas.o .libs/libglpk_la-asnhall.o .libs/libglpk_la-asnlp.o .libs/libglpk_la-asnokalg.o .libs/libglpk_la-ckasn.o .libs/libglpk_la-ckcnf.o .libs/libglpk_la-cplex.o .libs/libglpk_la-cpp.o .libs/libglpk_la-cpxbas.o .libs/libglpk_la-graph.o .libs/libglpk_la-gridgen.o .libs/libglpk_la-intfeas1.o .libs/libglpk_la-maxffalg.o .libs/libglpk_la-maxflp.o .libs/libglpk_la-mcflp.o .libs/libglpk_la-mcfokalg.o .libs/libglpk_la-mcfrelax.o .libs/libglpk_la-minisat1.o .libs/libglpk_la-mpl.o .libs/libglpk_la-mps.o .libs/libglpk_la-netgen.o .libs/libglpk_la-npp.o .libs/libglpk_la-pript.o .libs/libglpk_la-prmip.o .libs/libglpk_la-prob1.o .libs/libglpk_la-prob2.o .libs/libglpk_la-prob3.o .libs/libglpk_la-prob4.o .libs/libglpk_la-prob5.o .libs/libglpk_la-prrngs.o .libs/libglpk_la-prsol.o .libs/libglpk_la-rdasn.o .libs/libglpk_la-rdcc.o .libs/libglpk_la-rdcnf.o .libs/libglpk_la-rdipt.o .libs/libglpk_la-rdmaxf.o .libs/libglpk_la-rdmcf.o .libs/libglpk_la-rdmip.o .libs/libglpk_la-rdprob.o .libs/libglpk_la-rdsol.o .libs/libglpk_la-rmfgen.o .libs/libglpk_la-strong.o .libs/libglpk_la-topsort.o .libs/libglpk_la-weak.o .libs/libglpk_la-wcliqex.o .libs/libglpk_la-wrasn.o .libs/libglpk_la-wrcc.o .libs/libglpk_la-wrcnf.o .libs/libglpk_la-wript.o .libs/libglpk_la-wrmaxf.o .libs/libglpk_la-wrmcf.o .libs/libglpk_la-wrmip.o .libs/libglpk_la-wrprob.o .libs/libglpk_la-wrsol.o .libs/libglpk_la-btf.o .libs/libglpk_la-btfint.o .libs/libglpk_la-fhv.o .libs/libglpk_la-fhvint.o .libs/libglpk_la-ifu.o .libs/libglpk_la-luf.o .libs/libglpk_la-lufint.o .libs/libglpk_la-scf.o .libs/libglpk_la-scfint.o .libs/libglpk_la-sgf.o .libs/libglpk_la-sva.o .libs/libglpk_la-colamd.o .libs/libglpk_la-bfd.o .libs/libglpk_la-bfx.o .libs/libglpk_la-glpapi06.o .libs/libglpk_la-glpapi07.o .libs/libglpk_la-glpapi08.o .libs/libglpk_la-glpapi09.o .libs/libglpk_la-glpapi10.o .libs/libglpk_la-glpapi12.o .libs/libglpk_la-glpapi13.o .libs/libglpk_la-glpios01.o .libs/libglpk_la-glpios02.o .libs/libglpk_la-glpios03.o .libs/libglpk_la-glpios07.o .libs/libglpk_la-glpios09.o .libs/libglpk_la-glpios11.o .libs/libglpk_la-glpios12.o .libs/libglpk_la-glpipm.o .libs/libglpk_la-glpmat.o .libs/libglpk_la-glpscl.o .libs/libglpk_la-glpssx01.o .libs/libglpk_la-glpssx02.o .libs/libglpk_la-lux.o .libs/libglpk_la-alloc.o .libs/libglpk_la-dlsup.o .libs/libglpk_la-env.o .libs/libglpk_la-error.o .libs/libglpk_la-stdc.o .libs/libglpk_la-stdout.o .libs/libglpk_la-stream.o .libs/libglpk_la-time.o .libs/libglpk_la-tls.o .libs/libglpk_la-cfg.o .libs/libglpk_la-cfg1.o .libs/libglpk_la-cfg2.o .libs/libglpk_la-clqcut.o .libs/libglpk_la-covgen.o .libs/libglpk_la-fpump.o .libs/libglpk_la-gmicut.o .libs/libglpk_la-gmigen.o .libs/libglpk_la-mirgen.o .libs/libglpk_la-spv.o .libs/libglpk_la-minisat.o .libs/libglpk_la-avl.o .libs/libglpk_la-bignum.o .libs/libglpk_la-dimacs.o .libs/libglpk_la-dmp.o .libs/libglpk_la-ffalg.o .libs/libglpk_la-fp2rat.o .libs/libglpk_la-fvs.o .libs/libglpk_la-gcd.o .libs/libglpk_la-hbm.o .libs/libglpk_la-jd.o .libs/libglpk_la-keller.o .libs/libglpk_la-ks.o .libs/libglpk_la-mc13d.o .libs/libglpk_la-mc21a.o .libs/libglpk_la-mt1.o .libs/libglpk_la-mygmp.o .libs/libglpk_la-okalg.o .libs/libglpk_la-qmd.o .libs/libglpk_la-relax4.o .libs/libglpk_la-rgr.o .libs/libglpk_la-rng.o .libs/libglpk_la-rng1.o .libs/libglpk_la-round2n.o .libs/libglpk_la-spm.o .libs/libglpk_la-str2int.o .libs/libglpk_la-str2num.o .libs/libglpk_la-strspx.o .libs/libglpk_la-strtrim.o .libs/libglpk_la-triang.o .libs/libglpk_la-wclique.o .libs/libglpk_la-wclique1.o .libs/libglpk_la-mpl1.o .libs/libglpk_la-mpl2.o .libs/libglpk_la-mpl3.o .libs/libglpk_la-mpl4.o .libs/libglpk_la-mpl5.o .libs/libglpk_la-mpl6.o .libs/libglpk_la-mplsql.o .libs/libglpk_la-npp1.o .libs/libglpk_la-npp2.o .libs/libglpk_la-npp3.o .libs/libglpk_la-npp4.o .libs/libglpk_la-npp5.o .libs/libglpk_la-npp6.o .libs/libglpk_la-proxy.o .libs/libglpk_la-proxy1.o .libs/libglpk_la-spxat.o .libs/libglpk_la-spxchuzc.o .libs/libglpk_la-spxchuzr.o .libs/libglpk_la-spxlp.o .libs/libglpk_la-spxnt.o .libs/libglpk_la-spxprim.o .libs/libglpk_la-spxprob.o .libs/libglpk_la-spychuzc.o .libs/libglpk_la-spychuzr.o .libs/libglpk_la-spydual.o .libs/libglpk_la-adler32.o .libs/libglpk_la-compress.o .libs/libglpk_la-crc32.o .libs/libglpk_la-deflate.o .libs/libglpk_la-gzclose.o .libs/libglpk_la-gzlib.o .libs/libglpk_la-gzread.o .libs/libglpk_la-gzwrite.o .libs/libglpk_la-inffast.o .libs/libglpk_la-inflate.o .libs/libglpk_la-inftrees.o .libs/libglpk_la-trees.o .libs/libglpk_la-uncompr.o .libs/libglpk_la-zio.o .libs/libglpk_la-zutil.o   |  | /usr/bin/sed -e '/^[BCDGRS][ ]/s/.*[ ]\([^ ]*\)/\1 DATA/;s/^.*[ ]__nm__\([^ ]*\)[ ][^ ]*/\1 DATA/;/^I[ ]/d;/^[AITW][ ]/s/.* //' | sort | uniq > .libs/libglpk.exp
    [glpk-5.0.p1]   [spkg-install] ../libtool: eval: line 1103: syntax error near unexpected token `|'
    [glpk-5.0.p1]   [spkg-install] ../libtool: eval: line 1103: `/c/mingw64/bin/nm -B  .libs/libglpk_la-amd_1.o .libs/libglpk_la-amd_2.o .libs/libglpk_la-amd_aat.o .libs/libglpk_la-amd_control.o .libs/libglpk_la-amd_defaults.o .libs/libglpk_la-amd_dump.o .libs/libglpk_la-amd_info.o .libs/libglpk_la-amd_order.o .libs/libglpk_la-amd_post_tree.o .libs/libglpk_la-amd_postorder.o .libs/libglpk_la-amd_preprocess.o .libs/libglpk_la-amd_valid.o .libs/libglpk_la-advbas.o .libs/libglpk_la-asnhall.o .libs/libglpk_la-asnlp.o .libs/libglpk_la-asnokalg.o .libs/libglpk_la-ckasn.o .libs/libglpk_la-ckcnf.o .libs/libglpk_la-cplex.o .libs/libglpk_la-cpp.o .libs/libglpk_la-cpxbas.o .libs/libglpk_la-graph.o .libs/libglpk_la-gridgen.o .libs/libglpk_la-intfeas1.o .libs/libglpk_la-maxffalg.o .libs/libglpk_la-maxflp.o .libs/libglpk_la-mcflp.o .libs/libglpk_la-mcfokalg.o .libs/libglpk_la-mcfrelax.o .libs/libglpk_la-minisat1.o .libs/libglpk_la-mpl.o .libs/libglpk_la-mps.o .libs/libglpk_la-netgen.o .libs/libglpk_la-npp.o .libs/libglpk_la-pript.o .libs/libglpk_la-prmip.o .libs/libglpk_la-prob1.o .libs/libglpk_la-prob2.o .libs/libglpk_la-prob3.o .libs/libglpk_la-prob4.o .libs/libglpk_la-prob5.o .libs/libglpk_la-prrngs.o .libs/libglpk_la-prsol.o .libs/libglpk_la-rdasn.o .libs/libglpk_la-rdcc.o .libs/libglpk_la-rdcnf.o .libs/libglpk_la-rdipt.o .libs/libglpk_la-rdmaxf.o .libs/libglpk_la-rdmcf.o .libs/libglpk_la-rdmip.o .libs/libglpk_la-rdprob.o .libs/libglpk_la-rdsol.o .libs/libglpk_la-rmfgen.o .libs/libglpk_la-strong.o .libs/libglpk_la-topsort.o .libs/libglpk_la-weak.o .libs/libglpk_la-wcliqex.o .libs/libglpk_la-wrasn.o .libs/libglpk_la-wrcc.o .libs/libglpk_la-wrcnf.o .libs/libglpk_la-wript.o .libs/libglpk_la-wrmaxf.o .libs/libglpk_la-wrmcf.o .libs/libglpk_la-wrmip.o .libs/libglpk_la-wrprob.o .libs/libglpk_la-wrsol.o .libs/libglpk_la-btf.o .libs/libglpk_la-btfint.o .libs/libglpk_la-fhv.o .libs/libglpk_la-fhvint.o .libs/libglpk_la-ifu.o .libs/libglpk_la-luf.o .libs/libglpk_la-lufint.o .libs/libglpk_la-scf.o .libs/libglpk_la-scfint.o .libs/libglpk_la-sgf.o .libs/libglpk_la-sva.o .libs/libglpk_la-colamd.o .libs/libglpk_la-bfd.o .libs/libglpk_la-bfx.o .libs/libglpk_la-glpapi06.o .libs/libglpk_la-glpapi07.o .libs/libglpk_la-glpapi08.o .libs/libglpk_la-glpapi09.o .libs/libglpk_la-glpapi10.o .libs/libglpk_la-glpapi12.o .libs/libglpk_la-glpapi13.o .libs/libglpk_la-glpios01.o .libs/libglpk_la-glpios02.o .libs/libglpk_la-glpios03.o .libs/libglpk_la-glpios07.o .libs/libglpk_la-glpios09.o .libs/libglpk_la-glpios11.o .libs/libglpk_la-glpios12.o .libs/libglpk_la-glpipm.o .libs/libglpk_la-glpmat.o .libs/libglpk_la-glpscl.o .libs/libglpk_la-glpssx01.o .libs/libglpk_la-glpssx02.o .libs/libglpk_la-lux.o .libs/libglpk_la-alloc.o .libs/libglpk_la-dlsup.o .libs/libglpk_la-env.o .libs/libglpk_la-error.o .libs/libglpk_la-stdc.o .libs/libglpk_la-stdout.o .libs/libglpk_la-stream.o .libs/libglpk_la-time.o .libs/libglpk_la-tls.o .libs/libglpk_la-cfg.o .libs/libglpk_la-cfg1.o .libs/libglpk_la-cfg2.o .libs/libglpk_la-clqcut.o .libs/libglpk_la-covgen.o .libs/libglpk_la-fpump.o .libs/libglpk_la-gmicut.o .libs/libglpk_la-gmigen.o .libs/libglpk_la-mirgen.o .libs/libglpk_la-spv.o .libs/libglpk_la-minisat.o .libs/libglpk_la-avl.o .libs/libglpk_la-bignum.o .libs/libglpk_la-dimacs.o .libs/libglpk_la-dmp.o .libs/libglpk_la-ffalg.o .libs/libglpk_la-fp2rat.o .libs/libglpk_la-fvs.o .libs/libglpk_la-gcd.o .libs/libglpk_la-hbm.o .libs/libglpk_la-jd.o .libs/libglpk_la-keller.o .libs/libglpk_la-ks.o .libs/libglpk_la-mc13d.o .libs/libglpk_la-mc21a.o .libs/libglpk_la-mt1.o .libs/libglpk_la-mygmp.o .libs/libglpk_la-okalg.o .libs/libglpk_la-qmd.o .libs/libglpk_la-relax4.o .libs/libglpk_la-rgr.o .libs/libglpk_la-rng.o .libs/libglpk_la-rng1.o .libs/libglpk_la-round2n.o .libs/libglpk_la-spm.o .libs/libglpk_la-str2int.o .libs/libglpk_la-str2num.o .libs/libglpk_la-strspx.o .libs/libglpk_la-strtrim.o .libs/libglpk_la-triang.o .libs/libglpk_la-wclique.o .libs/libglpk_la-wclique1.o .libs/libglpk_la-mpl1.o .libs/libglpk_la-mpl2.o .libs/libglpk_la-mpl3.o .libs/libglpk_la-mpl4.o .libs/libglpk_la-mpl5.o .libs/libglpk_la-mpl6.o .libs/libglpk_la-mplsql.o .libs/libglpk_la-npp1.o .libs/libglpk_la-npp2.o .libs/libglpk_la-npp3.o .libs/libglpk_la-npp4.o .libs/libglpk_la-npp5.o .libs/libglpk_la-npp6.o .libs/libglpk_la-proxy.o .libs/libglpk_la-proxy1.o .libs/libglpk_la-spxat.o .libs/libglpk_la-spxchuzc.o .libs/libglpk_la-spxchuzr.o .libs/libglpk_la-spxlp.o .libs/libglpk_la-spxnt.o .libs/libglpk_la-spxprim.o .libs/libglpk_la-spxprob.o .libs/libglpk_la-spychuzc.o .libs/libglpk_la-spychuzr.o .libs/libglpk_la-spydual.o .libs/libglpk_la-adler32.o .libs/libglpk_la-compress.o .libs/libglpk_la-crc32.o .libs/libglpk_la-deflate.o .libs/libglpk_la-gzclose.o .libs/libglpk_la-gzlib.o .libs/libglpk_la-gzread.o .libs/libglpk_la-gzwrite.o .libs/libglpk_la-inffast.o .libs/libglpk_la-inflate.o .libs/libglpk_la-inftrees.o .libs/libglpk_la-trees.o .libs/libglpk_la-uncompr.o .libs/libglpk_la-zio.o .libs/libglpk_la-zutil.o   |  | /usr/bin/sed -e '/^[BCDGRS][ ]/s/.*[ ]\([^ ]*\)/\1 DATA/;s/^.*[ ]__nm__\([^ ]*\)[ ][^ ]*/\1 DATA/;/^I[ ]/d;/^[AITW][ ]/s/.* //' | sort | uniq > .libs/libglpk.exp'
    [glpk-5.0.p1]   [spkg-install] make[5]: *** [Makefile:600: libglpk.la] Error 2
    [glpk-5.0.p1]   [spkg-install] make[5]: Target 'all' not remade because of errors.
    [glpk-5.0.p1]   [spkg-install] Making all in examples
    [glpk-5.0.p1]   [spkg-install] gcc -DHAVE_CONFIG_H -I. -I..  -I./../src -I/c/a/passagemath/passagemath/sage-local/include   -g -O2 -c glpsol.c
    [glpk-5.0.p1]   [spkg-install] make[5]: *** No rule to make target '../src/libglpk.la', needed by 'glpsol.exe'.
    [glpk-5.0.p1]   [spkg-install] make[5]: Target 'all' not remade because of errors.
    [glpk-5.0.p1]   [spkg-install] make[4]: *** [Makefile:321: all-recursive] Error 1
    [glpk-5.0.p1]   [spkg-install] make[3]: *** [Makefile:252: all] Error 2
```

_Originally posted by @mkoeppe in https://github.com/passagemath/passagemath/issues/1194#issuecomment-3046234289_

--- a/build/pkgs/glpk/spkg-install.in
+++ b/build/pkgs/glpk/spkg-install.in
@@ -7,6 +7,8 @@ if [ -n "$MSYSTEM" ]; then
     for a in ../patches/mingw/*.patch; do
         patch -p1 < $a
     done
+    sed -i "s|-version-info|-no-undefined -version-info|g" src/Makefile.am
+    autoreconf -vfi
 fi
 
 # Note: The following doesn't work with spaces in `$SAGE_LOCAL`, but we don't

